docs/quickstart.rst

```diff
@@ -0,0 +1,16 @@
+ABC-Classroom Quickstart
+------------------------
+
+To set up a template directory to run abc-classroom in, run:
```

Comment:
We should advise people not to run this in an existing git repository (because the directory will eventually contain other git repos, and we want to avoid repos-in-repos).

Reply:
@nkorinek this could be a little breakout note!!

Reply:
i'm thinking we should do this but perhaps it's just a new issue so we can merge the functionality today!! i'd like to test everything together on master!

Reply:
https://github.com/earthlab/abc-classroom/issues/109 ok created... this can be done in a new PR

Reply:
actually i now see you already did this. resolving and closing that issue